test/ml_methods/test_sklearnMethod.py

```diff
@@ -59,3 +60,19 @@ def test_store(self):
         self.assertTrue(isinstance(svm2, SVC))
 
         shutil.rmtree(path)
+    #
```

Comment:
? :)

Reply:
fixed the bug that made this commented out test fail + added the test